lib/utils.bash

```diff
@@ -319,8 +323,15 @@ parse_asdf_version_file() {
   if [ -f "$file_path" ]; then
     local version
     version=$(strip_tool_version_comments "$file_path" | grep "^${plugin_name} " | sed -e "s/^${plugin_name} //")
+
     if [ -n "$version" ]; then
-      printf "%s\n" "$version"
+      local full_version="$version"
+      if [[ "$version" == path:* ]]; then
+        util_resolve_user_path "${version#path:}"
+        full_version="path:$REPLY"
```

Comment:
Same comment as above. I don't see a need for this `REPLY` variable as the resolved path could be printed.

Reply:
:+1: